POSHIT/Example CHT application/malaria-usecase-cht/targets.js

```diff
@@ -37,4 +37,45 @@ module.exports = [
     },
     date: 'reported'
   },
+  {
+    id: TARGETS.ALL_MEMBERS_WITH_MALARIA,
+    type: 'percent',
+    icon: 'child-under-5',
+    goal: -1,
+    translation_key: 'targets.all_members_with_malaria_symptoms',
+    subtitle_translation_key: 'targets.members.with.malaria.symptoms.subtitle',
+    appliesTo: 'contacts',
+    appliesToType: [CONTACT_TYPES.HOUSEHOLD_MEMBER],
+    appliesIf: function(c) {
+      return isContactValid(c);
+    },
+    passesIf: function(c) {
+      return c.reports.some(function(r) {
+        const under5 = r.form === FORMS.CHILD_ASSESSMENT && Utils.getField(r, 'children_under_assessment.suspected_of_malaria') === 'Yes';
+        const over5 = r.form === FORMS.HOUSEHOLD_MEMBER_ASSESSMENT && Utils.getField(r, 'group_household_member_assessment.suspected_of_malaria') === 'Yes';
+        return under5 && over5;
+      });
+    },
+    date: 'reported',
+  },
+  {
+    id: TARGETS.UNDER5_MEMBERS_WITH_MALARIA,
+    type: 'percent',
+    icon: 'child-under-5',
+    goal: -1,
+    translation_key: 'targets.u5_members_with_malaria_symptoms',
+    subtitle_translation_key: 'targets.under5_assessments_percentage.subtitle',
+    appliesTo: 'contacts',
+    appliesToType: [CONTACT_TYPES.HOUSEHOLD_MEMBER],
+    appliesIf: function(c) {
+      return ageInMonths(c.contact.date_of_birth) < 60 && isContactValid(c);
+    },
+    passesIf: function(c) {
+      ageInMonths(c.contact.date_of_birth) < 60;
```

Comment:
If you record the patient's age in the assessment report, you could use that value for this check.

The one downside I see with recalculating this age is that you could have some false negatives e.g if you were counting U5 members with malaria for the quarter and they ended up moving from the U5 bracket.